docs/write_tests.rst

```diff
@@ -0,0 +1,131 @@
+How to Write Tests for Parsons Connectors
+=========================================
+
+******************
+How to Write Tests
+******************
+
+Most of our Parsons Connector classes fall into two categories:
+
+ * Classes that call directly into HTTP API's using the Parsons ``APIConnector`` class
+ * Classes that wrap around third party Python libraries, often using their own "client" class
+
+The category that the class falls into generally determines how one will write unit tests for the class.  However, the category of Connector determines how we simulate the external API.
```

Comment:
I'm having trouble parsing this, specifically that second sentence. What are you trying to say here?